.github/workflows/testcases.yml

```diff
@@ -0,0 +1,22 @@
+name: running testCases
+
+on: [push]
```

Comment:
Why you didn’t copy from sample calculator ?
Where is the code review coverage ?

Reply:
the sample calculator made a lot of errors when i run it with mojo and jacoco so I did this sample at the beginning of the inter to use it and used integrated test coverage of intellij